test/integration/expand-composition.test.ts

```diff
@@ -67,7 +67,8 @@ describe('expand composition tokens', () => {
       start: ':root {',
       end: '--sdDeepRef: italic 800 26px/1.25 Arial;',
     });
-    const expectedOutput = `--sdCompositionSize: 24px;
+    // TODO: Why did this change?
```

Comment:
SD v5 refactored quite a few things about composite token expansion, optimizing it a lot. I suspect this is why the order of some tokens has changed because in the old SD we expanded tokens and then flattened, in the new SD we first flatten and then expand.